docs/reference/func_deploy.md

```diff
@@ -95,7 +95,7 @@ func deploy
   -p, --path string             Path to the project directory (Env: $FUNC_PATH) (default ".")
       --platform string         Target platform to build (e.g. linux/amd64).
   -u, --push                    Push the function image to registry before deploying (Env: $FUNC_PUSH) (default true)
-  -r, --registry string         Registry + namespace part of the image to build, ex 'ghcr.io/myuser'.  The full image name is automatically determined based on the local directory name. If not provided the registry will be taken from func.yaml (Env: $FUNC_REGISTRY)
+  -r, --registry string         Registry + namespace part of the image to build, ex 'ghcr.io/myuser'.  The full image name is automatically determined. (Env: $FUNC_REGISTRY) (default "image-registry.openshift-image-registry.svc:5000/demo")
```

Comment:
This change seems unrelated to the overall PR - and I don't think that the default in upstream is the openshift image registry.

Reply:
I have no idea how does this actually appeared here.

Reply:
Good catch, thanks. Fixed.